schematron/v2.2.0/exemplary_files/L200_Audit.xml

```diff
@@ -153,6 +159,49 @@
                   <auc:ThermalZoneLayout>Single zone</auc:ThermalZoneLayout>
                 </auc:Section>
 
+                <auc:Section ID="Section-Space-B">
+                  <auc:SectionType>Component</auc:SectionType>
+                  <auc:OccupancyClassification>Other</auc:OccupancyClassification>
```

Comment:
@corymosiman12 What should the auc:SectionType be for something like a Parking deck? We currently require any Space function to have linked lighting, hvac, etc, which I'm guessing we wouldn't want to for users to include.

This came up because I was trying to create a space that's excluded from the gross and conditioned floor area values (see above).

Reply:
Well, couldn't it still be `Space function` with `<auc:OccupancyClassification>Parking</auc:OccupancyClassification>`??

Good question though

Reply:
Ah yeah, I think I should have a different OccupancyClassification. I guess I'm trying to figure out how we know which of the Space functions we should make the extra assertions on

Reply:
Tracking here: https://github.com/BuildingSync/TestSuite/issues/42